docusaurus/docs/welcome/instructions/app-icon.md

```diff
@@ -24,14 +24,42 @@ Nonetheless, automatic generation should always be the first step.
 
 #### Android
 
-The Android assets can be generated using [Android Asset Studio](https://github.com/romannurik/AndroidAssetStudio).
+The Android assets can be generated using [Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio).
 
-The tool from the Android Asset Studio that generates the assets is the [Launcher icon generator](<https://romannurik.github.io/AndroidAssetStudio/icons-launcher.html#foreground.type=clipart&foreground.clipart=android&foreground.space.trim=1&foreground.space.pad=0.25&foreColor=rgba(96%2C%20125%2C%20139%2C%200)&backColor=rgb(68%2C%20138%2C%20255)&crop=0&backgroundShape=square&effects=none&name=ic_launcher>).
-There are some options which you can tweak as much as you like, but the only ones we care about are the "Shape" and "Name".
-You should create two versions of the app icon. The first one should have the "Square" shape and its "Name" should be "app_icon".
-The second's version shape should be the "Circle" and its "Name" should be "app_icon_round".
+To start the Asset Studio, follow these steps:
 
-> ℹ️ Although [Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio) was also considered, it isn't as intuitive and straightforward as the [Android Asset Studio](https://github.com/romannurik/AndroidAssetStudio).
+1. Open the project on Android Studio.
+2. In the **Project** window, select the [Android view](https://developer.android.com/studio/projects#ProjectFiles).
+3. Right-click the **res** folder and select **New > Image Asset**.
+
+![Image Asset Studio](/img/image-asset-studio.png)
+
+The app icon should have both a square and a round version.
+There tool offers a few options which you can tweak as much as you like, but we will only cover the necessary ones.
```

Comment:
```suggestion
There tool offers a few options which you can tweak as much as you like, but we only cover the necessary ones.
```

Reply:
I don't know why I wrote "There" this doesn't make sense! Will use your suggestion and fix this at the same time